pkg/logs/processor/encoder.go

```diff
@@ -108,19 +107,9 @@ func (r *raw) isRFC5424Formatted(content []byte) bool {
 type proto struct{}
 
 func (p *proto) encode(msg message.Message, redactedMsg []byte) ([]byte, error) {
-
-	// TODO Remove occurrences of "severity" (it is now "status")
-	// Compute the status
-	var status string
-	if msg.GetSeverity() != nil && bytes.Equal(msg.GetSeverity(), config.SevError) {
-		status = config.StatusError
-	} else {
-		status = config.StatusInfo
-	}
-
 	return (&pb.Log{
 		Message:   string(redactedMsg),
-		Status:    status,
+		Status:    message.SeverityToStatus(msg.GetSeverity()),
```

Comment:
Do we need the severity elsewhere? Can't we directly store the status in the message and forget about severity?